falcon/http_error.py

```diff
@@ -170,8 +173,7 @@ def to_json(self):
         """
 
         obj = self.to_dict(OrderedDict)
-        return json.dumps(obj, indent=4, separators=(',', ': '),
-                          ensure_ascii=False)
+        return json.dumps(obj, ensure_ascii=False)
```

Comment:
@kgriffs Developers probably pretty print the json on their own. (Via browser plugin or json_pp in bash). There is no need for it in a api.